core/scripts/umip-3/2_VoteSimulate.js

```diff
@@ -36,9 +44,9 @@ async function runExport() {
   if (argv.repeated) {
     assert((await governor.numProposals()).toNumber() > 1);
   } else {
-    assert.equal((await governor.numProposals()).toNumber(), 1); // there should be 1 proposal on the first run.
+    assert.equal((await governor.numProposals()).toNumber(), 2); // there should be 1 proposal on the first run.
```

Comment:
Should there still be 1 proposal at the start?

Reply:
there should be 2. have updated the comment accordingly.